jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -97,8 +97,8 @@ pipeline {
         )
         choice(
             name: 'DOCKER_SOURCE',
-            description: 'Enter the docker-source to pull the docker image, either DockerHub or ECR.',
-            choices: ["dockerhub", "ecr"]
+            description: 'Enter the docker-source to pull the docker image, either DockerHub or ECR or Both.',
+            choices: ["dockerhub", "ecr", "Both"]
```

Comment:
Why capital?

Reply:
It aligns with existing implementation https://github.com/opensearch-project/opensearch-build/blob/6508fcd795e2e0998ff760356f47bfcfc1bee776/jenkins/validate-artifacts/validate-artifacts.jenkinsfile#L103C8-L107C10

Reply:
Got it.